css-layout-classes/styles.css

```diff
@@ -0,0 +1,11 @@
+#body {
+  display: flex;
+  flex-direction: column;
+  margin: 0 10%;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  margin: 0 25px;
+}
```

Comment:
You should have multiple selectors for each of your `column` classes to handle how much space each column will take up in a row. You also should have a `row` class with `display: flex`